Sources/Views/EpisodePage.swift

```diff
@@ -685,17 +685,19 @@ private func mainContent(
 }
 
 private func downloadsView(episode: Episode) -> Node {
-  //.href(gitHubUrl(to: GitHubRoute.episodeCodeSample(directory: codeSampleDirectory)))
   .div(
     attributes: [
       .class([
-        Class.padding([.mobile: [.all: 3], .desktop: [.leftRight: 4, .bottom: 4, .top: 2]]),
-        ])
+        Class.padding([
+          .mobile: [.all: 3],
+          .desktop: [.left: 4, .right: 3, .bottom: 4, .top: 2]
+        ]),
+      ])
     ],
-    .h2(
+    .h3(
       attributes: [
         .id("downloads"),
-        .class([Class.h4, Class.type.lineHeight(3), Class.padding([.mobile: [.top: 2]])])
+        .class([Class.h3])
```

Comment:
titles are h3's in @amotion's newest designs, and these extra styles aren't needed